### ISSUE TITLE
Rétroaction.

### ISSUE THREAD

**Issue by CdeM-LucBergeron** [AlexisCottierGuilly/tp2#1]
- Ton programme ne demande pas les bornes au joueur.
- Erreur PEP8 à la fin : pas de 'enter' à la dernière ligne.
- Utilise des f-string pour les print.

**Comment by CdeM-LucBergeron** [comment]
Je vais enlever la pÉnalité pour l'erreur PEP8. Il semble que c'est une problématique avec le copier/coller à partir de Github. Je vais aussi enlever la pénalité pour les bornes.

**Comment by AlexisCottierGuilly** [comment]
Est-ce que pour les f-string c’est bon aussi ?

**Comment by CdeM-LucBergeron** [comment]
> Est-ce que pour les f-string c’est bon aussi ?

Oui! Désolé. Finalement ton code est parfait :) Ça te donne 100%
